lib/modules/platform/types.ts

```diff
@@ -87,13 +87,14 @@ export interface Issue {
   title?: string;
 }
 export type PlatformPrOptions = {
-  azureAutoApprove?: boolean;
-  azureWorkItemId?: number;
-  bbUseDefaultReviewers?: boolean;
-  gitLabIgnoreApprovals?: boolean;
+  azureAutoApprove: boolean;
+  azureWorkItemId: number;
+  bbUseDefaultReviewers: boolean;
+  gitLabIgnoreApprovals: number;
```

Comment:
? why number? shouldn't we convert the bool option to corresponding number value at platform code?

Reply:
i think they should stay optional and the platform code should handle it